packages/components/MenuList/components/MenuListInner.js

```diff
@@ -133,9 +133,9 @@ const MenuListInnerComponent = ({
   const menuList = [
     {
       id: isPersonal ? personalLists.inbox?.id : null,
-      title: 'Inbox',
+      title: 'Shared with me',
```

Comment:
In the middle column is still Inbox title.
Maybe fix it in selector 'inboxListSelector'?

Reply:
@cu4xxi in the code the menuList has priotity over the server data, idkw. Btw, works for me.

Reply:
@beshkenadze 
you're talking about menuList.
But I am about MiddleColumn.
I've reproduce such case with favorites (see screenshot)
https://scr.4xxi.com/cu/Dashboard_2020-10-15_3_PM-29-19.png